docs/design-document.md

```diff
@@ -64,76 +73,192 @@ Repository
 ```
 
 
-### Actors
+### Actors and Authentication
 
-In a Git repository, an "actor" is any user who makes changes to the repository.
-These changes can involve any part of the repository, such as modifying files,
-branches or tags. In the gittuf system, each actor is identified by a unique
-signing key that they use to sign their contributions. This means that when a
-policy allows an actor to make certain changes, it's actually allowing the
-person who has the specific signing key to make those changes. To maintain
-security, all actions made in the repository, such as adding or modifying files,
-are checked for authenticity. This is done by verifying the digital signature
-attached to the action, which must match the public key associated with the
-actor who is supposed to have made the change.
+In a Git repository, an "actor" is any party, human or bot, who makes changes to
+the repository. These changes can involve any part of the repository, such as
+modifying files, branches or tags. In gittuf, each actor is identified by a
+unique signing key that they use to cryptographically sign their contributions.
+gittuf uses cryptographic signatures to authenticate actors as these signatures
+can be verified by anyone who has the corresponding public key, fundamental to
+gittuf's mission to enable independent verification of repository actions. Note
+that gittuf does not use Git commit metadata (e.g., author email, committer
+email) to identify the actor who created it as that may be trivially spoofed.
+
+In practice, when a gittuf policy allows an actor to make certain changes, it's
+actually allowing the person who has the specific signing key to make those
```

Comment:
should 'person' -> 'identity'?   If so, don't just make the change because identities aren't known to the reader.   

Also, you talk about 'actor' before this and 'person' here?   Is an 'actor' a 'person'?   Do we need two terms?

Reply:
actor is generic, can be person or bot. This shouldn't really say "person" here though.

Reply:
rephrased to avoid confusion.